config/initializers/inflections.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
```

Comment:
![image](https://github.com/AyoMoses1/Gems_Blog/assets/93039359/cd2e843c-ccc5-48d3-b0bd-768865b4e6c7)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍